### ISSUE TITLE
Create Praktisk info section

### ISSUE THREAD

**Issue by niccofyren** [gathering/tgno-backend#8]
# General

The Praktisk section is where information that is helpful to participants (and potential participants) live. It should be structured and easy to navigate, and users/editorial will adjust hierarchy as needed. It is not the place for information trying to "sell" TG in any way.

# Relevant unknowns

While we have an impression of what the information stored here will be (see sitemap), we do not know the best way of exposing and managing that information to users visiting the site. Focus on using native Wagtail page behaviour and structure in first iteration so that we have a neutral starting point that is easy to understand and work with.

# Endpoint

Should follow regular Wagtail REST Api setup and live under `.../info/`

# Data structure

It is probably quite safe to start with the structure we know from our existing page. Once frontend is in place we can adjust as needed which fields are shown, and how they are used (or removed).

Fields we should expect in first version
- Body
- Intro
- Title
- Main image
- Publishing fields (we should get these from Wagtail out of the box, think last edited, scheduling, etc)

Fields we should avoid for now
- Side column items* (facts, menu, related content, ...)

\* New page is intended to be less hands-on. Meaning we want to expose less of these type of options to users initially, and instead rely on fixed logic for what shows up and where.

# Presentation

Out of scope in this initial backend implementation. But in short it will probably be
- Individual info pages with links to related items
- Navigation that makes it easy to navigation to and read about a certain topic
- Some sort of main page where key information is shown (likely somewhat adjustable by users)
- Some sort of information on individual pages indicating if they have been updated for the "next relevant year" or might be out of  date
